.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "sdk/go-sdk"]
+	path = sdk/go-sdk
+	url = git@github.com:layotto/go-sdk.git
```

Comment:
It looks like this file is no longer needed.

Reply:
Done.